docs/processors/standalone/index.mdx

```diff
@@ -4,8 +4,92 @@ sidebar_position: 3
+
+## How to write one
+
+Thanks to our [Web Assembly (Wasm) processor](/docs/processors/standalone/how-it-works) you can start writing processors in any language that can be compiled to Web Assembly. As a start, Conduit already provides a [conduit-processor-sdk](https://github.com/ConduitIO/conduit-processor-sdk) that will let you [write a processor](/docs/processors/standalone/building) in Golang.
+
+### Where to put them
+
+By default, standalone processors are expected to be found in a folder named `processors` alongside of your pipelines or [standalone connectors](/docs/connectors/getting-started.mdx):
+
+```shell
+│ # Conduit binary
+├── conduit
+│ # Folder with pipeline configurations
+└── pipelines
+│ # Folder with standalone connectors
+└── connectors
+│ # Folder with standalone processors
+└── processors
+```
+
+However, in case you need to reference processors in a different location, you could use the `-processors.path` flag when running Conduit:
+
+```
+./conduit -processors.path /my-custom-processors-path
+```
+
+### Using the [conduit-processor-sdk](https://github.com/ConduitIO/conduit-processor-sdk)
+
+Assuming you use our [conduit-processor-sdk](https://github.com/ConduitIO/conduit-processor-sdk), this is how a processor **plugin** written in Golang could look like. In the following example, we're going to be adding a `processed` field to each record processed by our pipeline:
+
+```go
+//go:build wasm
+
+package main
+
+import (
+    "context"
+
+    "github.com/conduitio/conduit-commons/opencdc"
+    sdk "github.com/conduitio/conduit-processor-sdk"
+)
 
-🚧 Page is under construction, content will be added soon.
+func main() {
+    sdk.Run(sdk.NewProcessorFunc(
+        sdk.Specification{Name: "simple-processor", Version: "v1.0.0"},
+        func(ctx context.Context, record opencdc.Record) (opencdc.Record, error) {
+            record.Payload.After.(opencdc.StructuredData)["processed"] = true
+            return record, nil
+        },
+    ))
+```
 
+After that, you'd need to compile it, and locate its `.wasm` file into the desired `processors` directory as previously mentioned:
+
+
+```shell
+GOARCH=wasm GOOS=wasip1 go build -o simple-processor.wasm main.go
+```
+
+## Using it in your pipeline
+
+
+As mentioned in our [Getting Started page](/docs/processors/getting-started), in order to use a processor in your pipeline, you need to update its configuration file [accordingly](/docs/processors/getting-started#how-to-use-a-processor) and [reference it](/docs/processors/referencing) accordingly:
+
+
+```yaml
+version: 2.2
+pipelines:
+  - id: example-pipeline
+    connectors:
+    # define source and destination connectors
+    # ...
+    processors:
+      - id: add-processed-field
+        plugin: standalone:simple-processor
+```
+
+:::warning
+When refering a processor, you need to make sure each one of them has its own name and version even if they're using the same processor plugin.
```

Comment:
I'm not sure what this is describing 😕 Do you mean that IDs need to be unique? Because you can reuse the same processor plugin (i.e. name and version) in a pipeline.

Or are you describing the fact that if you have multiple custom processor plugins they need to have a unique name+version? If so, then that is not an issue with referring to the processor but rather developing it, so I'd move it up to where you show how to write the simple processor.

Reply:
@lovromazgon oh yeah, I was confused about these two (using vs building). You're right. I'll move it elsewhere.